Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb

```diff
@@ -402,7 +424,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.6"
+   "version": "3.6.8"
```

Comment:
Due to the way that azure notebooks work (the system I'm using to deliver the lectures), this actually needs to be 3.6.6 for now.  Might be good to create an environment with that specific version of Python.